source/crud/tests/db/db-aggregate.json

```diff
@@ -0,0 +1,147 @@
+{
+  "data": [],
+  "tests": [
+    {
+      "description": "Aggregate with $currentOp",
+      "operation": {
+        "name": "db-aggregate",
+        "database_name": "admin",
+        "arguments": {
+          "pipeline": [
+            {
+              "$currentOp": {
+                "allUsers": true,
+                "idleConnections": true
+              }
+            },
+            {
+              "$match": {
+                "command.aggregate": {
+                  "$eq": 1
+                }
+              }
+            },
+            {
+              "$project": {
+                "command": 1
+              }
+            },
+            {
+              "$project": {
+                "command.lsid": false
+              }
+            }
+          ]
+        }
+      },
+      "outcome": {
+        "result": [
+          {
+            "command": {
```

Comment:
This looks like an APM expectation. Is this really an aggregation result due to the database-level aggregation being the only in-progress operation at the time it is executed and therefore it only reports itself?

Reply:
Yes, it is an aggregation result that is only reporting itself.